src/main/resources/view/OrderCard.fxml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.VBox?>
+
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+  <GridPane HBox.hgrow="ALWAYS">
+    <columnConstraints>
+      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
+    </columnConstraints>
+    <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
+      <padding>
+        <Insets top="5" right="5" bottom="5" left="15" />
+      </padding>
+      <HBox spacing="5" alignment="CENTER_LEFT">
+        <Label fx:id="id" styleClass="cell_big_label">
+          <minWidth>
+            <!-- Ensures that the label text is never truncated -->
+            <Region fx:constant="USE_PREF_SIZE" />
+          </minWidth>
+        </Label>
+        <Label fx:id="cheeseType" text="\$first" styleClass="cell_big_label" />
+      </HBox>
+      <Label fx:id="name" styleClass="cell_small_label" text="\$name" />
+      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
+      <Label fx:id="quantity" styleClass="cell_small_label" text="\$quantity" />
+      <Label fx:id="orderDate" styleClass="cell_small_label" text="\$orderDate" />
+      <Label fx:id="completedDate" styleClass="cell_small_label" text="\$completedDate" />
+    </VBox>
```

Comment:
Is it possible to increase the padding? The cards seems a bit too packed

![image](https://user-images.githubusercontent.com/12421332/111058796-dadbd680-84cb-11eb-992d-831d20bfa8b4.png)

Reply:
Noted, will address this before tmr (monday) morning